tools/wpt.ts

```diff
@@ -539,7 +539,25 @@ function discoverTestsToRun(
     for (const key in parentFolder) {
       const entry = parentFolder[key];
 
-      if (Array.isArray(entry)) {
+      const expectation = Array.isArray(parentExpectation) ||
+          typeof parentExpectation == "boolean"
+        ? parentExpectation
+        : parentExpectation[key];
+
+      if (
+        Array.isArray(entry) &&
+        key.endsWith(".html") &&
+        typeof expectation === "boolean"
+      ) {
```

Comment:
Makes it possible to run .html files directly.

Reply:
any reason why not opening a PR upstream to change the tests to js? its what we have done until now

Reply:
This should be easier going forward, right?